### ISSUE TITLE
Criminal reputation mechanics

### ISSUE THREAD

**Issue by FJonske** [better-modernized-combat/bmod-client#141]
I don't know if this one needs a plugin, but I think it might.

RIght now, Criminals are pretty limited in where they can go, as once they go red with House or Civilian factions, they are attacked on sight. After discussion with @IrateRedKite, we've come to the conclusion that it might be nice to have some mechanic in place that allows criminals to effectively hide their identity, similar to how real criminals aren't just ganked by every pedestrian, but only by the police, and only once they are positively identified.

Options for this include:
- Capping the rep for civilian and house factions at the lowest end of white rep and modifying the player's cargo or something similar with an arrest warrant - if an appropriate faction member scans you, all of the capped factions turn hostile to you.
- A sort of cloaking or hacking mechanic that changes your rep very temporarily for a small price or inconvenience (waiting while a hack completes, flying without cruise in cloak, etc), so you can slip through a docking area without being turned into space debris.

**Comment by IrateRedKite** [comment]
Closing as a duplicate of #133
